src/m365/spo/commands/page/page-control-set.ts

```diff
@@ -31,10 +32,24 @@ class SpoPageControlSetCommand extends SpoCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
-    Page
-      .getPage(args.options.name, args.options.webUrl, logger, this.debug, this.verbose)
-      .then((clientSidePage: ClientSidePage): Promise<ClientSidePageProperties> => {
-        const control: ClientSidePart | null = clientSidePage.findControlById(args.options.id);
+    const requestOptions: any = {
+      url: `${args.options.webUrl}/_api/SitePages/Pages/GetByUrl('sitepages/${encodeURIComponent(args.options.name)}')`,
```

Comment:
This call fails if the page name doesn't contain `.aspx` extension. We should check the page name specified by the user and append the extension if it wasn't specified.